plugins/modules/net_tools/nmcli.py

```diff
@@ -1492,9 +918,8 @@ def show_connection(self):
                             alias_key = alias_pair[0]
                             alias_value = alias_pair[1]
                             conn_info[alias_key] = alias_value
-                elif key in ['ipv4.dns', 'ipv4.dns-search', 'ipv6.dns', 'ipv6.dns-search']:
-                    values = raw_value.split(',')
-                    conn_info[key] = values
+                elif key_type == list:
```

Comment:
The operator `is` is preferred for checking type objects compared to the operator `==`.

For example, `elif key_type is list:`

Reply:
Fixed in 766433ef.